src/private/Utils_p.h

```diff
@@ -126,7 +126,7 @@ inline bool usesUtilityWindows()
 
 inline bool isNormalWindowState(Qt::WindowStates states)
 {
-    return !(states & (Qt::WindowMaximized | Qt::WindowMinimized | Qt::WindowFullScreen));
+    return !(states & (Qt::WindowMaximized | Qt::WindowFullScreen));
```

Comment:
please add this to the first commit, so we don't have a do and a redo in the history

maybe just sqush the whole commit with the first